src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,19 +15,21 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream().filter(w -> w.length() > 12).count();
```

Comment:
```suggestion
        return words.stream().filter(StreamStudy::isLongWord).count();
```

로직에 대한 의미를 담은 메소드로 분리시켜볼수도 있을것 같아요 😄